SAKPaaS/src/app/components/user-card/user-card.component.html

```diff
@@ -5,10 +5,16 @@
     <div class="expand">
       <mat-icon (click)="change()" class="expand-icon">keyboard_arrow_left</mat-icon>
     </div>
+    
+    <div *ngIf="expanded">
+      <a href="https://auth.happyhamster.org/auth/realms/happyhamster/account/">
```

Comment:
```suggestion
      <a href="https://auth.happyhamster.org/auth/realms/happyhamster/account/" title="title="{{'sidenav.settings' | translate}}"">
```

and in addition pls add it to the text-symbols. :)

Reply:
That copy'n'taste error I did here. ^^

Reply:
Already fixed